docs/content/doc/upgrade/from-gogs.en-us.md

```diff
@@ -19,9 +19,10 @@ Gogs, version 0.9.146 and older, can be easily migrated to Gitea.
 
 There are some basic steps to follow. On a Linux system run as the Gogs user:
 
-* Create a Gogs backup with `gogs dump`. This creates `gogs-dump-[timestamp].zip` file
-  containing all important Gogs data.
+* Create a Gogs backup with `gogs backup`. This creates `gogs-backup-[timestamp].zip` file
+  containing all important Gogs data. You would need it if you wanted to move to the `gogs` back later.
 * Download the file matching the destination platform from the [downloads page](https://dl.gitea.io/gitea).
+ It should be `1.0.x` version. Migrating from `gogs` to any other version is impossible.
```

Comment:
(nitpicking) could say instead that it is *recommended* users use gitea 1.0.

Reply:
That means you could upgrade twice for less pain.

Gogs -> Gita 1.0.x -> <Gitea recent version>

Reply:
I'm scratching my head now... I thought I saw someone say they did Gogs -> Gitea 1.3, but probably because I read 1.0.3 as 1.3.

Reply:
@charlesreid1 I tried both `1.3.x` and `1.4.x` versions, and probably even `1.2.x` - neither worked except `1.0.x` version.

Reply:
Great - that's valuable info and glad it's going into the docs!

Reply:
@charlesreid1 nevermind, everyone should help each other :) Is there anything else I can do in this PR?

Reply:
I'm good to go, think it just needs to be merged now.